spec/requests/api/v1/participant_declarations_spec.rb

```diff
@@ -55,9 +55,11 @@ def build_params(attributes)
       end
 
       it "create declaration record and declaration attempt and return id when successful" do
-        expect { post "/api/v1/participant-declarations", params: build_params(valid_params) }
+        params = build_params(valid_params)
+        expect { post "/api/v1/participant-declarations", params: params }
             .to change(ParticipantDeclaration, :count).by(1)
             .and change(ParticipantDeclarationAttempt, :count).by(1)
+        expect(ApiRequestAudit.last.body).to eq(params.to_s)
```

Comment:
Implicit ordering with `last`, is it correct?

Reply:
Well, there should only be one, but I fixed it.